src/main/java/com/vkatit/cinema/service/Printable.java

```diff
@@ -0,0 +1,11 @@
+package com.vkatit.cinema.service;
+
+import com.vkatit.cinema.model.Ticket;
+import org.springframework.stereotype.Service;
+
+@Service
+public interface Printable {
```

Comment:
if it's printable then method should be print not getTicket. It would be reasonable that service support not only pdf but images formats as was mentioned in jira.

Reply:
Method "getTicket” changed to method “print”
Implementation of both PDF and images is possible
(implementation “PdfGeneratorService” has been removed since it is no longer used in the Controller “download” method)